packages/web-client/app/utils/web3-strategies/layer1-chain.ts

```diff
@@ -312,4 +319,11 @@ export default abstract class Layer1ChainWeb3Strategy
       this.networkSymbol
     )}/${txnHash}`;
   }
+
+  async getEstimatedGasForWithdrawalClaim(): Promise<BN> {
+    let gasPrice = await this.web3!.eth.getGasPrice();
+    return APPROXIMATE_WITHDRAWAL_GAS_AMOUNT.mul(
```

Comment:
@habdelra @Aierie I changed this a bit. This is actually the version shown in the Loom I made. Let me know what you think.

Reply:
I think the rounding is inconsistent. 

```
let fourteen = new BN('14');
let thirteen = new BN('13');
let four = new BN('4');

fourteen.divRound(four); // 4
thirteen.divRound(four); // 3
```

which means that we might sometimes floor rather than ceil and make the gas price look lower than our actual estimate. I'm not sure how likely this is to cause issues though.

Reply:
Might also have issues w amounts smaller than the rounder (7 decimal places)

Reply:
We have already rounded up the gas amount (290,000) so I think this is safe.